### ISSUE TITLE
particles/antiparticles names

### ISSUE THREAD

**Issue by brauliorivas** [key4hep/dmx#12]
I have a doubt. Every particle ID is associated with a name. So for example, $d$ quark has an ID 1. This is based on the [montecarlo scheme](https://pdg.lbl.gov/2007/reviews/montecarlorpp.pdf). However, as explained in the last meeting, every particle has its antiparticle. So $d$ quark with ID 1, has the antiquark $\bar{d}$ with ID = -1. 
I understand this logic but don't know where it comes from. The cited paper shows the particles but not the antiparticles. In dmX, many particles have their equivalent antiparticle definition. I would like to know more about this because many newly incorporated particles lack their antiparticle.

**Comment by tmadlener** [comment]
Hi Braulio, IIUC, your main question is where the negative values for the anti particles come from? This is just by convention, the pdf says that it only shows the particles, and that it is assumed for the rest of the document, the main pat is: 

> This encodes information about the particle’s spin, flavor content, and
> internal quantum numbers. The details are as follows:
> 1. Particles are given positive numbers, antiparticles negative
> numbers. The PDG convention for mesons is used, so that K+
> and B+ are particles.

In principle the lookup could also just use the absolute value to go for the general particle, and then use the sign to switch it to the anti particle. However, for some particles that involves more than just putting a bar on top of it.

**Comment by brauliorivas** [comment]
> However, for some particles that involves more than just putting a bar on top of it.

Right! So how do I add the antiparticle? For example, the particle's name with ID = 23 is $Z^0$. But how do I add the antiparticle name? I know that the ID would be = -23, but I don't know how would the name be. (I want to have the LaTeX equivalent, mainly known as _name_).

**Comment by kjvbrt** [comment]
In your example the Z is it's own antiparticle, meaning ID = -23 will be also $Z^0$. The same for $\gamma$
